index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Isaac Jung's Resume</title>
+    <link rel='stylesheet' href='styles.css' />
+</head>
+
+<body>
+    <div class="page u-clearfix">
+        <main>
```

Comment:
I expect you can simplify things by removing this `div` and instead adding your class names directly to the `main` element.